rwengine/src/engine/SaveGame.cpp

```diff
@@ -1327,29 +1331,37 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 #ifdef RW_WINDOWS
 char* readUserPath() {
     LONG retval;
-    const char* lpSubKey = "Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Shell Folders";
+    const char* lpSubKey =
```

Comment:
nit: `constexpr` :p

Reply:
That won't change a thing for a `const char*`.

Reply:
I was thinking more about `static constexpr char const*`. :P